integration/confidential/fixtures/offline-fs-kbc/s390x/aa-offline_fs_kbc-resources.json

```diff
@@ -1,5 +1,8 @@
jhPTlRvdWxyYTlhUFJGWApHZHJnQkdRQkduWkhnSWxUYmpibTdiRTZFUjlwd05tUmFPRVMrcFRtRWxWTzMzTVJ4alUxOEFEK3Z4TEp3MXcwCmprb1dQUmlxQnk0T0huU1JhZ0lpT1h5S3NwQ1Z3WTF3dkYzMjY3VCttYWxId2Z5OUp3SDBxdytBbTU3L3pGbWUKSkZwdGNPeEVkMXVqNThyZzcyZ1FNQzVYZG9wWVVmd281aVZKa1d6ZU1PL1RlTHIxOE1venhoczJ4eENGVG5xTApCZXJ1NnhVTm9TQXZxMGVxNnRJb3VCekhZZlF2WUV2Ri9ueVZ1TEZFYU95RzJZTVhjVWlBVUV0eHVRTktIbFFXCm11ZXRaSmdlQkoxNU9TREtmTVQ5VUtWdGIyaHJPYkVBU013UGV3ZytTVDRZODcrSnN4OUY2Zlo0NTVvZFBhOWQKbFhpYnBkRzBCL2oxbXhVQUVRRUFBYlFuU0hsdmRXNW5aM2wxSUVOb2Iya2dQRWg1YjNWdVoyZDVkUzVEYUc5cApRR2xpYlM1amIyMCtpUUhVQkJNQkNnQStGaUVFcE9NZUwvM3V4cUlPSjhISHE0NVdxem5yRldNRkFtTjJYZ0VDCkd3TUZDUVBDWndBRkN3a0lCd0lHRlFvSkNBc0NCQllDQXdFQ0hnRUNGNEFBQ2drUXE0NVdxem5yRldQUFlndisKT2dFQUFwakpsMWNlMjVadHpPVWRsNXdBS0p0T0RNNTMrVFRpQkRhRm9MWWxZNVg1R1NsRy96dHpsaDdldzhVeApDcjVGeVppNHNoT3FvQ3h3cmQ3aHNHbmJwVkR1ZDRLU0lqdmFBaGYvZm52ZUd5K1lyNFdJaXNrZnJaZXFDREFWCnpPV29UZU9SMFRQSjFhUFlzRWgveDQrT3dFYkdId1E2N01sMkVuYXdYNjdNWG1NTDdnQ3ZiR2lDdFkxV3JYN3QKUktqWlVLKzN0SWZYNHBFZ3AxUW1VQlVlOWxNUzNMc0YxYXRTdEtwRDZqbHRwamtxSXhnd0ZvTThsNkh0Q0o2MgpYamdDTzAzcHUrcVpWUUJzdnZEb2V4dWEvaXB4MWZxVUFpekY3dlEzYlB1WjNJQ1F0ZUtXaXI1S0RQTEtqcG9QCldObGxCU1d4b0VwK0lndWNSMy9lUGJYekpGelVKM2taaS96cmFPNlpBdW1QZlJSamtjS2JiK3dJVVp6VVhmR2YKVmFxaGE4b0lmYVgzWG9JUDJDYkpYdG1jVU53V2hpT05UL21QMXhaTXdEblI4K1ZjaGtyY1dReFE4OHJxZnh1RQp0a2liTDZyU0hJV1BvVFVaUmNleVlNN3B1a1gyQVFxY0ZjWTV6c2tVUGZlb29FaDdRdXVyZEhuRnhCZDVGMS9hCnVRR05CR04yWGdFQkRBRFBNcGx6RkpGRytpZmlnd1JnT2ZJVEdXWkxIV0ZqMWxaM1lrZ2dHWHo4L1Q1bGhwd3AKTXZMTHpWUDJZSm14Q0hqSDFjNGFJQ3FBelVXbjJBOENWVlpKbnlTaVVQTy9kS1Nna3ZtYW1rUkd4Q2ZwWUxObApHTXE4dng3elBhdG1UUjM1RVMxWlM3bE9UN0FRaVRPRHdhTmdsbFNhU2lhc3hmc2UvS09UQnZIblovT0N0b1ZXCmtuMlFlTjgya2l2ZGh5MGVMZFY1REdVVm9OdTlhQ2J4c1d4U2tMT1E3d1d6aENsb3k1ajJrZ3psZUNYbUsxdmMKMVJUTlptNUFSSG03dHpHMTdVRzd2eW5Ua1hQbVIvcmlkR0k1YlJvcStKeUkrWUh4dm9iaTlxZkVRT3lKVk1JNgpMb294ZlVaWkFEN0UzWkkwdzhld296UmtWV2tnRktzSmVVTko5RzVZK1RFSFdSUG45TlVFNzEyVmxvNWhVT04zCkNUQ0N2ek5CQmcycTRHQ1VFMno4WG1QRkRwMzJMZ1d1em9HWVJVdEh3SmVKTmhZQWtGMVpIYmJDczF0UExkTWIKTzZJaW8zVVVnRHFmRy9TOVJpOGhJTHcyVmx3eVBFQSs1YitXd2JUbTBiL3ZPMUUvcDVvRlFLaDNvd1MrcVNMbgo4TnQ5Y2pQSGJDNUIwL0VBRVFFQUFZa0J2QVFZQVFvQUpoWWhCS1RqSGkvOTdzYWlEaWZCeDZ1T1ZxczU2eFZqCkJRSmpkbDRCQWhzTUJRa0R3bWNBQUFvSkVLdU9WcXM1NnhWajNqb0wrd1lUVy9BU3hGRXVGUWJvZm1PbTh5OGEKUUl2TWpiRDBnQUlTM0Y5QytxUnlSUjYyN1dPak5jSGl3RVA5YUh3SVZFRXZiaVJkZk9lNVV2Z2M4NlpjaEFhMwo1Yk9XR3VKQ0JIWU5GNU9RcUpxTXlvWmpmRWRFSEpWVW9wbGhOd3NUa0V3MGc3ZXU0TEpkMWxsN0UvVVpDWmVvClhudlJxTVFEUjRUV2ZYeks5dzZONXNrZHJXcHZWeGNCNEUwVW9BVmh2alVNalo2S2ZmUGYrOS9SWTRLdG83S0EKbXVhZUkxV0M4RGRjZUJnZDg5QXNhMVVOWCtpVC9DVGw0endQWXQzMjVjcmhIVG5YZ05BRFo4QXZhUUFyc24wWQpGT0svcGJ0aUczOEtkalJBQjNlN3dDTFdWaVV4QkhBQ1lyUzkvUjBxVzRCa1Y5OHNwZXRSVWxhYlVycDVFbHNaClNXTVFpNk95NzlYRkZMam55eS9JbzdGSDREdXlKQTlLZ3lyaDFJUXU0c3ZWeWhaTWVGbW5GUkdkSUVqM2JBcW0Kbm9ZSlA2WElTeUFGNjB1UEJteEdidXpvTm9CbUVKZDFHeTVBZ0ZyQ1p3K1gxc1ZtTVZUMit3YVpvK0lINjlGVwpoNW1hdERZdDh2VE1zRmxuaEN5NllmUDdQcEhaQmNBa1JXYU9SNkVrL1E9PQo9aWpVOQotLS0tLUVORCBQR1AgUFVCTElDIEtFWSBCTE9DSy0tLS0tCg=="
```

Comment:
I think this should be `default/gpg-public-config/test` rather than `default/sigstore-config/test`

Reply:
Done